src/mainwindow.cpp

```diff
@@ -121,6 +121,33 @@ MainWindow::MainWindow():
   setShowThumbnails(settings.showThumbnails());
   ui.actionShowThumbnails->setChecked(settings.showThumbnails());
 
+  addAction(ui.actionMenubar);
+  ui.menubar->setVisible(settings.isMenubarShown());
+  ui.actionMenubar->setChecked(settings.isMenubarShown());
+  connect(ui.actionMenubar, &QAction::triggered, this, [this](int checked) {
+    if(!isFullScreen()) { // toolbar is hidden in fullscreen
```

Comment:
You mean "*menubar* is hidden in fullscreen".

Reply:
yes... copy/paste... will fix it.